russstation/code/modules/mob/living/carbon/emote.dm

```diff
@@ -22,6 +22,8 @@
 
 /datum/emote/living/carbon/fart/check_cooldown(mob/user, intentional)
 	var/mob/living/carbon/U = user
+	if(issilicon(user))
+		return
```

Comment:
This is being called on `mob/living/carbon`s, so by definition the user shouldn't be a `mob/living/silicon`, and therefore this is redundant.

Reply:
With added fart burn since this comment, silicons will get human fart messages if silicon exclusion isn't specified. If there's another way to prevent human messages for them, I can remove the if issilicon.